server/lib/chat/chat-models.ts

```diff
@@ -48,13 +48,18 @@ export interface UserChannelsEntry {
   joinDate: Date
 }
 
+export interface ChatMessageData {
```

Comment:
This isn't a correct typing imo, this is only the form for one particular type (we may only have one type atm, but the way you have typed this does not indicate that we expect there to be more in the future and that adding more forms is acceptable/expected)